File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Gui/App/Models/Templates/TemplateCollaborationRepresentationType.ts
enum TemplateCollaborationRepresentationType {
    /** 
     * This are special type of fixed records (types of tempalte collaboration) are are
     * at Level1, under the root and they can have folders and tempaltes as children
     */
    FixedTemplates = 0,

    /**
     * Representing a template and the tree edge node. The node can have no more children.
     **/
    Template = 1,

    /**
     * These are folders that can have only Folders and templates as children
     **/
    Folder = 2
}

export = TemplateCollaborationRepresentationType;

File: SOURCE_WEBAPI/VecompSoftware.DocSuiteWeb.Entity/Templates/TemplateCollaborationRepresentationType.cs
namespace VecompSoftware.DocSuiteWeb.Entity.Templates
{
    public enum TemplateCollaborationRepresentationType : short
    {
        /// <summary>
        /// This are special type of fixed records (types of tempalte collaboration) are are 
        /// at Level1, under the root and they can have folders and tempaltes as children
        /// </summary>
        FixedTemplates = 0,

        /// <summary>
        /// These cannot have children
        /// </summary>
        Template = 1,

        /// <summary>
        /// These are folders that can have only Folders and templates as children
        /// </summary>
        Folder = 2
    }
}
